src/components/cax-companyData.tsx

```diff
@@ -206,7 +206,7 @@ export const CompanyDataCax = () => {
   }
 
   const onSearchChange = (expr: string) => {
-    if (isBPN(expr)) {
+    if (isBPN(expr?.trim())) {
```

Comment:
if we follow the discussed convention to always trim any input value I think it's better to do the trim inside the `isBPN` (and all other similar) functions so the code here would be better readable.

Reply:
Would be unexpected behavior for isXYZ functions imho. You check a value against REGEX. Trimming here is only valid because we take care of white spaces somewhere else.

Reply:
okay - there are different ways to solve that. Anyway the trim has to go somewhere. Thinking about different input fields the question becomes more generic and tricky anyway: for example let's consider spaces inside some values like IBAN or phone numbers and how the system should deal with those. From a user perspective i guess it is less annoying if the system automatically parses and reformats values instead of showing an error like when pasting IBANs, phone numbers, etc. which are common to be written with spaces.